src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -605,6 +662,11 @@ public String getQuestionResultStatisticsCsv(
                 String percentageFrequency = df.format(rubricStats[i][j] * 100) + "%";
                 csv.append("," + percentageFrequency + " (" + responseFrequency[i][j] + ")");
             }
+
+            if (hasAssignedWeights) {
+                csv.append("," + dfAverage.format(rubricStats[i][numOfRubricChoices]));
```

Comment:
Ok, got it. We can leave it for now, but it looks like it should be done as 2 methods, one for the percentage frequencies and one for average.